lib/Command/Configure.php

```diff
@@ -221,7 +222,31 @@ private function configureSms(InputInterface $input, OutputInterface $output) {
 
 				$providerConfig->setUrl($url);
 				break;
+				
+			case 'android_gsm_modem':
+				$config->setProvider($provider);
+				
+				/** @var AndroidGsmModemConfig $providerConfig */
+				$providerConfig = $config->getProvider()->getConfig();
+				
+				$hostQuestion = new Question('Please enter your GSM Modem IP address (e.g: x.x.x.x:8090)');
+				$host = $helper->ask($input, $output, $hostQuestion);
+				
+				$usernameQuestion = new Question('Please enter your GSM Modem API username: ');
+				$username = $helper->ask($input, $output, $usernameQuestion);
+
+				$passwordQuestion = new Question('Please enter your GSM Modem API password: ');
+				$password = $helper->ask($input, $output, $passwordQuestion);
 
+				$protocolQuestion = new Question('Please enter your GSM Modem API protocol (http/https): ');
```

Comment:
let's remove this and use only a *URL*, username and password. The URL can then be a domain or IP, http or https and also have a custom port. That is the most flexible and easiest to configure :)